### PR TITLE
Feature#4 register view

### PR DESCRIPTION
**Pull request by Shadic78**
# Description
What did you implemented/Why did you implemented this:
 - Implemented RegisterView Screen without functionality
 - Implemented TextBox component
 - Implemented CustomButton component
 - implemented CheckBoxWithLabel component

  ## Screenshots 
image

image

### CODE REVIEW

File: src/components/TextBox.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+const styles = StyleSheet.create({
+  textField: {
+    borderWidth: 1,
+    borderColor: '#5c5c5c',
+    marginBottom: 15,
+    paddingLeft: 12,
+  },
+});
+
```
Comment (YilKanda) [inline src/components/TextBox.jsx:24]: The same from the previous comment.

File: src/screens/RegisterView.jsx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  label: {
+    fontSize: 16,
+    alignSelf: 'center',
+    color: '#4d4d4d',
+  },
+  sectionIn: {
+    color: '#5c6ef8',
+    textDecorationLine: 'underline',
+  }
+});
```
Comment (YilKanda) [inline src/screens/RegisterView.jsx:94]: The same from the previous comment of the styles.

**Review by YilKanda — CHANGES_REQUESTED** [review]

File: src/components/CheckBoxWithLabel.jsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  checkBoxText: {
+    alignSelf: 'center',
+    fontSize: 16,
+    color: '#4d4d4d',
+  },
+  checkBoxContainer: {
+    alignContent: 'center',
+    flexDirection: 'row',
+  },
+});
```
[the remark this answers is not in the document]
Comment (Shadic78) [reply src/components/CheckBoxWithLabel.jsx]: I made the changes.

**Review by YilKanda — APPROVED** [review]
LGTM!

**Review by DemisRincon — DISMISSED** [review]

**Review by jkzasori — COMMENTED** [review]
Please, resolve the conflicts and try again.

**Review by robermancera — APPROVED** [review]

**Review by 96vladi — APPROVED** [review]

**Review by ljau — APPROVED** [review]

**Review by jkzasori — APPROVED** [review]

**Pull request merged**